airbyte-integrations/connectors/destination-s3-data-lake/src/test-integration/kotlin/io/airbyte/integrations/destination/s3_data_lake/S3DataLakeTestUtil.kt

```diff
@@ -24,22 +24,33 @@ object S3DataLakeTestUtil {
             ValidatedJsonUtils.parseOne(S3DataLakeSpecification::class.java, Files.readString(path))
         )
 
-    fun getAWSSystemCredentials(): AWSSystemCredentials {
-        val configFile = GLUE_AWS_ASSUME_ROLE_CONFIG_PATH.toFile()
-        return Jsons.readValue(configFile, AWSSystemCredentials::class.java)
+    fun getAwsAssumeRoleCredentials(): AwsAssumeRoleCredentials {
+        val creds = getAwsAssumeRoleCredentialsAsMap()
+        val assumeRoleAccessKey = creds["AWS_ACCESS_KEY_ID"]!!
+        val assumeRoleSecretKey = creds["AWS_SECRET_ACCESS_KEY"]!!
+        val assumeRoleExternalId = creds["AWS_ASSUME_ROLE_EXTERNAL_ID"]!!
```

Comment:
Nit: use the constants over these strings

Reply:
:+1: I'll do that in a later PR since this PR is earlier in git history

Reply:
done in second commit of https://github.com/airbytehq/airbyte/pull/52093